PRINCIPLES.md

```diff
@@ -0,0 +1,13 @@
+# Presto Foundation Principles
+
+## 1) One open, neutral and united Presto community 
+
+One of the main reasons the founding members came together and created the Presto Foundation was to unite the Presto community under the original project. The Presto community and it’s contributors have been working on many great capabilities and connectors, and many of these haven’t made their way back into the repo, and we truly believe that these can benefit many other users. We invite all Presto community members to embrace PrestoDB and contribute back to move the project faster, reduce duplication of efforts and create wider impact for the entire community.
+
+## 2) Open, transparent technical leadership and direction
+
+At the end of the day, technology and technical direction is what keeps a community interested and the innovation going. We want the project to be inclusive and the technical roadmap to be fairly representative of the community’s view. In fact, the TSC has its own open charter and the meetings are open for everyone to attend. We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges. 
+
+## 3) No one individual or company is greater than the team and project 
```

Comment:
team and project -> the project and its community

Reply:
(To me it genal if we are using this articulate he principle. )
We invite you to join the TSC meetings and we believe under the leadership of our TSC Chairperson Nezih Yigitbai (PrestoDB Lead at Facebook), the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.  
-> 
We invite you to join the TSC meetings. We believe under the open and transparent governance of TSC, the PrestoDB project will continue to grow, gain contributors, and solve some hard technical challenges.